utils/devel/README.md

```diff
@@ -1,87 +1,52 @@
-# 
-Tools to assist LedgerSMB developers
+# Tools to assist LedgerSMB developers
 
-######Copyright (c) 2015 ledgersmb.org
+<!-- markdownlint-disable MD001 -->
+<!-- markdownlint-disable MD018 -->
+###### Copyright (c) 2015 ledgersmb.org
 
     Licensed under the GPLv2 a copy of which can be found in /LICENSE
 
-For more information about any of these files, Read The Source Luke
+    For more information about any of these files, Read The Source Luke
```

Comment:
Drop the 4 preceding spaces as they cause the text to be shown in a codeblock
Instead, possibly make it Bold and Underlined

Reply:
Codeblock removed